.travis.yml

```diff
@@ -0,0 +1,9 @@
+language: c
+dist: trusty
```

Comment:
Any reason to use a VM and not a container based build? I think only if we'd have tests that would install and try to use a client/server we would need root.
Also don't we need 'sudo: required' when 'dist: trusty' is specified?